Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -24,6 +25,8 @@ namespace
 	const FString AverageClientRTTMetricName = TEXT("UnrealAverageClientRTT");
 	const FString AverageClientViewLatenessMetricName = TEXT("UnrealAverageClientViewLateness");
 	const FString PlayersSpawnedMetricName = TEXT("UnrealActivePlayers");
+	const FString AverageFPSValid = TEXT("UnrealFPSValid");
```

Comment:
add server, useful for indicating that this will be testing the offloaded server in multi-worker also